Cargo.toml

```diff
@@ -452,6 +452,7 @@ members = [
 	"substrate/utils/prometheus",
 	"substrate/utils/wasm-builder",
 ]
+default-members = [ "polkadot" ]
```

Comment:
Will this also build the needed execution worker binaries?

Reply:
Yes, they are all binaries of the `polkadot` package. The only thing broken here is `cargo run` (which would only build `polkadot` binary but not workers), but it's been broken for some time anyway.

Reply:
We should also set default run

Reply:
Makes sense. I believe it selects the first binary anyway, but it's always good to be explicit.